components/post_markdown/system_message_helpers.tsx

```diff
@@ -422,7 +422,7 @@ export function renderSystemMessage(post: Post, currentTeam: Team, channel: Chan
     return null;
 }
 
-function renderReminderACKMessage(post: Post, currentTeam: Team, isMilitaryTime: boolean): ReactNode {
+function renderReminderACKMessage(post: Post, currentTeam: Team, isMilitaryTime: boolean, timeZone?: string): ReactNode {
```

Comment:
Nit: Can we standardize on `timezone` wherever possible in this PR? Looks like we use that more consistently in this codebase:
```
λ~/w/mattermost-webappλ rg timeZone | wc -l
81
λ~/w/mattermost-webappλ rg timezone | wc -l
545
```

Looks like the prop of the 3rd party FormattedDate is `timeZone`, so not saying try and change that.

Reply:
Done